### ISSUE TITLE
Black theme settings window sizes don't match with other themes

### ISSUE THREAD

**Issue by Kuddus73** [AbdurazaaqMohammed/AntiSplit-M#105]
The hight and width of the settings window of the black theme don't with other themes.

**Black** theme:
![Screenshot_20241007_194629_AntiSplit M](https://github.com/user-attachments/assets/af495f71-a39b-4011-99b2-420dba34e8c0)

**Dark** theme:
![Screenshot_20241007_194625_AntiSplit M](https://github.com/user-attachments/assets/1cdc0fb2-f948-4ce3-974d-0f92c301dd0a)

**Light** theme:
![Screenshot_20241007_195328_AntiSplit M](https://github.com/user-attachments/assets/f62538c9-2f38-4dc5-9e9f-fe5b2f048b8e)

**Can you reduce dark theme's settings window sizes and match them with other themes?**

**Comment by drogga** [comment]
Check ticket 63 (I'm not linking it on purpose, you can subscribe to it to follow the bug reports, instead of creating a new issue for every single minor or not thing).
TL;DR: It's a known issue, caused by the border (dev. can't fix it for now).

**Comment by AbdurazaaqMohammed** [comment]
I add dialog border in all themes and unify dialog size 
![WhatsApp Image 2024-10-07 at 17 18 12_fd26852d](https://github.com/user-attachments/assets/b4d1b1d3-00b3-443d-8064-e89cd56b4171)
![WhatsApp Image 2024-10-07 at 17 22 45_83d5a6fb](https://github.com/user-attachments/assets/606d7a31-2dd2-45df-8bc3-63bd5bea2b38)

**Comment by Kuddus73** [comment]
> Check ticket 63 (I'm not linking it on purpose

@drogga The dev wasn't aware of this bug I reported as it was a new bug unrelated to the issue you _purposely didn't link_ and they already released a fix, so you clearly didn't know what you were talking about.

> instead of creating a new issue for every single minor or not thing

**It's only up to the dev to complain about the way of reporting** (which they didn't) and, if necessary, provide us guidelines or suggestions to follow when reporting bugs.
Also, you might be new to GitHub but it's good practice to open a new issue if it is different from the already opened ones, **even if it is a minor one** _or not thing_.

_TL;DR:_ In the future, avoid commenting just for the sake of writing something.

**Comment by drogga** [comment]
Dear Kuddus73, My comment was an FYI one and based on the text by the dev., which is the following: "_I added it, now the dialog takes up the entire screen for some reason and I couldn't fix it, this is the only way I got it to work_", which is the exact thing you've reported here and it wasn't only for the Settings dialog.
I'm not new, nagging, trying pick fights, being sarcastic or anything, so this isn't even a thing to which I can say something like "don't be a snowflake" (if you know what it means) and I'm reporting bugs and give ideas, suggestions and whatever way before you in another repo regarding not only this app by this very same user, ever since the first ver. and he can confirm.
I don't type in vain, just for the sake of writing something. I'm only trying to be helpful.

If you have issues with my comments, then I can delete them.

**Comment by Kuddus73** [comment]
> I'm not new, nagging, trying pick fights, being sarcastic or anything, so this isn't even a thing to which I can say something like "don't be a snowflake" (if you know what it means)

And yet in the very next sentence you mention "being a snowflake", _which obviously means you are not trying to pick fights..._
Well, you definitely do know the meaning of snowflake since you have written an essay of response.

> I'm reporting bugs and give ideas, suggestions and whatever way before you in another repo

Wow, congrats on being the first and still failing to figure out how GitHub reporting works.
Anyway, do as you please or whatever but ignore my reports and comments, as I will do the same with yours.
